monkey/infection_monkey/exploit/mssqlexec.py

```diff
@@ -153,7 +153,7 @@ def _brute_force(
 
         logger.warning(
             "No user/password combo was able to connect to host: {0}:{1}, "
-            "aborting brute force".format(host, port)
+            "aborting brute force".format(host.ip, port)
```

Comment:
`host` is already an IP... this is what happens when people neglect good naming conventions